apps/web/src/stores/apiKeys.ts

```diff
@@ -10,7 +10,7 @@ import useSWR, { SWRConfiguration } from 'swr'
 export default function useApiKeys(opts?: SWRConfiguration) {
   const { toast } = useToast()
   const key = 'api/apiKeys'
-  const fetcher = useFetcher(ROUTES.api.apiKeys.root)
+  const fetcher = useFetcher<ApiKey[]>(ROUTES.api.apiKeys.root)
```

Comment:
Lot of changes in this PR are related with improving types in `useFecher`. I wanted to extract what is inside and also being typed independent of the store.